perma_web/requirements.in

```diff
@@ -95,4 +95,6 @@ watchdog                                # watch for file changes
 pip-tools
 
 # instrumentation
-sentry-sdk
\ No newline at end of file
+sentry-sdk
+
+dict2xml
```

Comment:
This is tiny, but I think I would put this somewhere up higher in `requirements.in`, either in the general section, or maybe better in link preservation under `internetarchive`?